apps/desktop-wallet/package.json

```diff
@@ -13,8 +13,7 @@
   "scripts": {
     "build": "tsc && cross-env INLINE_RUNTIME_CHUNK=false VITE_VERSION=$npm_package_version vite build",
     "start": "turbo start:electron",
-    "start:web": "cross-env VITE_VERSION=$npm_package_version vite",
-    "start:electron": "concurrently \"pnpm start:web\" \"electron .\"",
```

Comment:
no need to run `electron .` anymore. The vite plugin takes care of it. We just need to run `vite`.